src/webhook/get-payload.ts

```diff
@@ -0,0 +1,26 @@
+
+import { Request } from "express";
+
+export function getPayload(request: Request): Promise<object> {
```

Comment:
preferred function styles as `export const getPayload.....`

Reply:
maybe we add func-style to eslint?

Reply:
probably a good idea @joshkay10.  Tag, you're it :P

Reply:
**Dodges** .... Unsuccessfully - https://github.com/atlassian/github-for-jira/pull/1226

Reply:
Removed file as now using middleware to parse request body.